src/components/Post/PostCard.tsx

```diff
@@ -0,0 +1,103 @@
+import { cn } from "@/utils/cn";
+import { formatTimeRange } from "@/utils/datetime";
+import IconLocation from "@/assets/icon/location.svg?react";
+import IconTime from "@/assets/icon/time.svg?react";
+import IconArrow from "@/assets/icon/arrow-up.svg?react";
```

Comment:
```suggestion
import { Location, Time, ArrowUp } from "@/assets/icon";
```
수빈님께서 일괄 export를 해주셔서 위 처럼 아이콘을 사용할 수 있어요! 👍